CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] 2021-12-14
+### Added
+- `PrivateKey` and `PublicKey` types, offering the following functions:
+```go
+GenerateKey(rnd io.Reader) (sk *PrivateKey, err error) // to generate a new keypair
+(sk *PrivateKey) Public() PublicKey // To export a public key from a private
+(sk *PrivateKey) Bytes() // To serialise a private key
+(sk *PrivateKey) Prove(alpha []byte) (beta, proof []byte, err error) // To generate proof and VRF
+(pk PublicKey) Verify(alpha, proof []byte) (verified bool, beta []byte, err error) // To verify proofs and VRFs
```

Comment:
I think we should make it a struct now then, so that we already have all the helpers

Reply:
I mean, we can also make the helpers without having it be a struct, no?

Reply:
Yes, but then some people might instantiate it directly, since it needs to be exported. This would be a breaking change if we when we make it a struct